app/screens/thread/thread.tsx

```diff
@@ -40,6 +41,8 @@ const styles = StyleSheet.create({
     flex: {flex: 1},
 });
 
+const EMPTY_STRING = '';
```

Comment:
very weird.. make channelId optional in the component that is receiving this

Reply:
I already made it optional; I forgot about these changes. Done it, corrected it.